lib/powerpoint/views/text.xml.erb

```diff
@@ -0,0 +1,28 @@
+<p:sp>
+<p:nvSpPr>
+  <p:cNvPr id="<%= id %>" name="Content Placeholder 2"/>
+  <p:cNvSpPr>
+    <a:spLocks noGrp="1"/>
+  </p:cNvSpPr>
+  <p:nvPr>
+    <p:ph idx="<%= idx %>"/>
+  </p:nvPr>
+</p:nvSpPr>
+<p:spPr/>
+<p:txBody>
+  <% unless bullets %>
+  <a:bodyPr>
```

Comment:
More of the same nit. Unless it borks the XML somehow, would prefer to see indentation within conditionals in these views. They're so dense already, proper indentation would make it easier to find & follow what logic is present.